test/partisan_SUITE.erl

```diff
@@ -824,3 +868,33 @@ make_certs(Config) ->
        {certfile, filename:join(PrivDir, "client/keycert.pem")},
        {cacertfile, filename:join(PrivDir, "client/cacerts.pem")}
       ]}].
+
+%% @private
+check_forward_message(Node, Manager) ->
+    {ok, Members} = rpc:call(Node, Manager, members, []),
+    %% ask member node to forward a message to one other random member
+    ct:pal("members of ~p: ~p", [Node, Members]),
+    RandomMember = random(Members, Node),
+    ct:pal("requesting node ~p to forward message to store_proc on node ~p",
+           [Node, RandomMember]),
+    Rand = rand_compat:uniform(),
+    rpc:call(Node, Manager, forward_message,
```

Comment:
i guess that's possible, if that happens the test will fail either way since the subsequent `application:get_env` won't return the expected value